controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -988,6 +1033,20 @@ func (r *BareMetalHostReconciler) getHostStatusFromAnnotation(host *metal3v1alph
 	return objStatus, nil
 }
 
+// extract host from HardwareDetails annotation
```

Comment:
nit: is this comment out of date?

Reply:
Hmm, not sure how this happened tbh, probably a copy/paste/sed typo, will update to clarify